package.json

```
@@ -349,6 +349,7 @@
     "@types/parse-torrent": "^5.8.3",
     "@types/react-router-dom": "^5.1.9",
     "@types/webtorrent": "^0.109.0",
+    "@solana/web3.js": "1.36.0",
```

```suggestion
    "@solana/web3.js": "^1.36.0",
```

Was there an intention here for version locking or can we use caret convention here as well? Looks like that's what is most commonly used in this file.

---

https://github.com/brave/brave-core/pull/12554#discussion_r824113329

---

Thanks for pointing out that comment. I think that's a well justified reason to do this and am happy to leave this version pinned based on that rationale.